src/app/views/organization/organization.html

```diff
@@ -16,6 +16,16 @@ <h1 translate>mnoe_admin_panel.dashboard.organization.organization_info</h1>
           </div>
           <div class="bs-row row" ng-if="::main.adminConfig.isPaymentEnabled()">
             <div class="col-xs-6">
+              <div class="label-cell" translate>mnoe_admin_panel.dashboard.organization.account_frozen</div>
+            </div>
+            <div class="col-xs-6">
```

Comment:
@x4d3 This is nested under the `ng-if="::main.adminConfig.isPaymentEnabled()"`
So it doesnt display on frontend where payment is disabled (Kappa)